### PR TITLE
feat: Add ControlExtensions.AlternateContent and MaterialToggleButtonIconStyle

### PR DESCRIPTION
**Pull request by agneszitte-nventive**
GitHub Issue: https://github.com/unoplatform/uno.ui.toolkit/issues/81

## PR Type

What kind of change does this PR introduce?

- Feature

## Description

Add ControlExtensions.AlternateContent and MaterialToggleButtonIconStyle

## PR Checklist 
Please check if your PR fulfills the following requirements:

- Commits must be following the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [X] Tested UWP
- [X] Tested iOS
- [X] Tested Android
- [X] Tested WASM
- [ ] Tested MacOS
- [X] Contains **No** breaking changes

### CODE REVIEW

**Review by davidjohnoliver — APPROVED** [review]
add to the list of things that need proper documentation ;)

**Review by kazo0 — APPROVED** [review]

File: src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/Content/Extensions/ControlExtensionsSamplePage.xaml [hunk 1]
```diff
@@ -9,6 +9,10 @@
...
+		<x:String x:Key="Icon_more_vertical">M 2 4 C 3.100000023841858 4 4 3.100000023841858 4 2 C 4 0.8999999761581421 3.100000023841858 0 2 0 C 0.8999999761581421 0 0 0.8999999761581421 0 2 C 0 3.100000023841858 0.8999999761581421 4 2 4 Z M 2 6 C 0.8999999761581421 6 0 6.899999976158142 0 8 C 0 9.100000023841858 0.8999999761581421 10 2 10 C 3.100000023841858 10 4 9.100000023841858 4 8 C 4 6.899999976158142 3.100000023841858 6 2 6 Z M 2 12 C 0.8999999761581421 12 0 12.899999976158142 0 14 C 0 15.100000023841858 0.8999999761581421 16 2 16 C 3.100000023841858 16 4 15.100000023841858 4 14 C 4 12.899999976158142 3.100000023841858 12 2 12 Z</x:String>
+	</Page.Resources>
 
 	<ScrollViewer VerticalScrollBarVisibility="Auto">
 		<StackPanel BorderBrush="{StaticResource MaterialOnSurfaceBrush}"
```
Comment (Xiaoy312) [inline src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/Content/Extensions/ControlExtensionsSamplePage.xaml:18]: Please adding spacing between sections. Ideally wrap each section in a stackpanel:
```xml
<StackPanel Spacing="16">
			
	<!-- ControlExtensions.Icon-->
	<StackPanel Spacing="16">
		<TextBlock Style="{StaticResource SomeHeaderStyle}" />

		<!-- IconElement Examples -->
		<StackPanel><!-- text + 4icons --></StackPanel>

		<!-- Supported Controls -->
		<StackPanel><!-- text + (text+control)*2 --></StackPanel>
	</StackPanel>

	<!-- ControlExtensions.AlternateContent-->
	<StackPanel>
		<TextBlock Style="{StaticResource SomeHeaderStyle}" />
		<!-- text + controls -->
	</StackPanel>
</StackPanel>
```

File: src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/Content/Extensions/ControlExtensionsSamplePage.xaml [hunk 2]
```diff
@@ -104,6 +106,24 @@
...
 				</ComboBox>
 			</smtx:XamlDisplay>
+
+			<TextBlock Text="ControlExtensions.AlternateContent"
+					   Style="{StaticResource MaterialHeadline5}" />
+
+			<TextBlock Text="This property can be used in order to have an alternate content for ToggleButton for example:"
+					   Style="{StaticResource MaterialBody1}" />
+
+			<smtx:XamlDisplay UniqueKey="ControlExtensionsSamplePage_ToggleButton"
```
Comment (Xiaoy312) [inline src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/Content/Extensions/ControlExtensionsSamplePage.xaml:116]: We could have different varieties here, like: (text: "Dark mode" vs "Light mode")

**Review by Xiaoy312 — APPROVED** [review]

File: src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/Content/Extensions/ControlExtensionsSamplePage.xaml (hunk 2 above)
Comment (agneszitte-nventive) [reply src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/Content/Extensions/ControlExtensionsSamplePage.xaml:116]: good idea, will add an example for it

**Pull request merged**